app/javascript/controllers/mobile_displayer_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static get targets() {
+    return ['menu', 'body'];
+  }
+
+  toggleHomeMenu(_) {
```

Comment:
@harold93 no jquery pretty please :)

Reply:
Also, this is brittle. This relies on the fact that all states (initial and after) match the selector interactions.
I found more stable to read from the html (as the source of truth) the result of the interaction.